components/shared/link-component.tsx

```diff
@@ -1,35 +1,35 @@
 'use client'
-import React, { ReactNode } from 'react'
 
-import classNames from 'clsx'
+import { HTMLAttributes } from 'react'
+
 import Link from 'next/link'
```

Comment:
If you want to keep the name as Link instead of LinkComponent you could do this
```suggestion
import { default as NextLink } from "next/link";
```

Reply:
Thanks for the suggestion, in this case, we want to keep it named `LinkComponent`, since it has a different logic from the plain `Link` from next and it is good to have a clear difference between both when using it.good catch!